components/support/utils/src/test/java/mozilla/components/support/utils/URLStringUtilsTest.kt

```diff
@@ -114,6 +114,10 @@ class URLStringUtilsTest {
         assertFalse(isURLLike("http://www-.com"))
         assertFalse(isURLLike("www.c-c-  "))
         assertFalse(isURLLike("3-3 "))
+
+        // Examples from issues
+        assertTrue(isURLLike(
+            "https://disposable-wide4--solidsoft-blog-disposable.netlify.app/")) // fenix/10747
```

Comment:
Let's remove the fenix ticket references. I'd use a generic example and link to the A-C issue.